packages/ng-primitives/popover/src/popover-trigger/popover-trigger.ts

```diff
@@ -213,7 +224,13 @@ export class NgpPopoverTrigger<T = null> implements OnDestroy {
     }
 
     // Show the overlay
-    this.overlay()?.show();
+    this.overlay()
+      ?.show()
+      .then(() => {
+        if (this.open()) {
+          this.opened.emit();
+        }
+      });
```

Comment:
I think we should switch this to use async/await - it would be better for the consumer that they can await the method to know when it has actually opened - I should have done this before 😊

```suggestion
    await this.overlay()?.show();
    this.openChange.emit(true);
```